modeltranslation/fields.py

```diff
@@ -400,6 +447,27 @@ def __get__(self, instance, owner):
         return None
 
 
+class TranslatedManyToManyDescriptor(object):
+    """
+    A descriptor used to return correct related manager without language fallbacks.
+    """
+
+    def __init__(self, field_name, fallback_languages):
+        self.field_name = field_name  # The name of the original field
+        self.fallback_languages = fallback_languages
+
+    def __get__(self, instance, owner):
+        # TODO: do we really need to handle fallbacks with m2m relations?
```

Comment:
I'm fine with leaving them without fallbacks, but we must document reasoning behind this choice.

Reply:
Whatever you see fitting. Caveats is fine.